modules/articles/client/config/articles.client.routes.js

```diff
@@ -12,22 +12,51 @@ angular.module('articles').config(['$stateProvider',
       })
       .state('articles.list', {
         url: '',
-        templateUrl: 'modules/articles/client/views/list-articles.client.view.html'
+        templateUrl: 'modules/articles/client/views/list-articles.client.view.html',
+        controller: 'ArticlesListController'
       })
       .state('articles.create', {
         url: '/create',
         templateUrl: 'modules/articles/client/views/create-article.client.view.html',
+        controller: 'ArticlesController',
```

Comment:
Well previously I had them all separate controllers but that got shot down.  I can see the need for a create controller if we don't want to pass in a blank article to resolve.  Then the ArticlesController would handle edit/view since it needs an article to resolve?  @rhutchison